docs/proposals/16-adminkubeconfig-subresource.md

```diff
@@ -96,7 +96,7 @@ status:
         client-key-data: LS0tLS1CRUd...
 ```
 
-The `AdminKubeconfigRequest` feature gate (enabled by default starting from `v1.39`) enables the above mentioned API in the `gardener-apiserver`. The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.
+The `AdminKubeconfigRequest` feature gate (enabled strictly starting from `v1.42`) enables the above mentioned API in the `gardener-apiserver`. The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.
```

Comment:
Nit: as this is the proposal doc, I would rather remove a statement about the feature gate state. The suitable thing for the proposal is to make it clear that the functionality was introduced behind a feature gate.

```suggestion
New feature gate called `AdminKubeconfigRequest` enables the above mentioned API in the `gardener-apiserver`. The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.
```